apache/trafficcontrol
# Validate DS's ORG server's cachegroup with topology's cachegroup

## What does this PR (Pull Request) do?




## Which Traffic Control components are affected by this PR?


- Traffic Ops

## What is the best way to verify this PR?

Method 1:
1. Assign ORG servers to a DS
2. Assign a topology which has origin cachegroup as one of the nodes.
3. Now, remove origin cachegroup from a given DS's topology and it should not allow you to do it.

Method2:
1. Remove ORG server part of DS1. 
2. Assign Topo1 to DS2 (which has ORG servers). 
3. Go to DS1's, manage topo and remove ORG CG from topo1. It should still error since topo1 is part of DS2, which has ORG server assignment.

Also, run the api/v3 test cases.

## If this is a bug fix, what versions of Traffic Control are affected?


## The following criteria are ALL met by this PR




## Additional Information

API route hasn't changed. Hence the documentation is not updated.
